### DISCUSSION TITLE
Debugging Modules

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1309] [General]
Not sure if this is a known or documented but to build and debug my modules simply, I add the Oqtaned Server project to my Module solutions and set as startup project and hit F5. No more attaching the debugger! 
[Studio-Elf Debugging](https://studio-elf.ga/mynews?Aid=17)

**Comment by sbwalker** [comment]
This approach definitely simplifies the development/debugging effort. Does your module still use the debug,cmd to deploy the module assemblies to the Oqtane /bin - or is this not necessary with your approach?

**Reply by leigh-pointer** [reply]
@sbwalker yes everything is the same. I also modified a copy of the templates so that its added when i create a new module.

**Comment by hishamco** [comment]
What you did is make sense and sure it simplify the debugging process but still there's a lack in WebAssembly debugging
